staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -513,3 +530,91 @@ func (u unionTransformers) TransformFromStorage(ctx context.Context, data []byte
 func (u unionTransformers) TransformToStorage(ctx context.Context, data []byte, dataCtx value.Context) (out []byte, err error) {
 	return u[0].TransformToStorage(ctx, data, dataCtx)
 }
+
+// DynamicTransformers holds transformers that are dynamically updated.
+type DynamicTransformers struct {
+	transformTracker *atomic.Value
+}
+
+type transformTracker struct {
+	transformerOverrides map[schema.GroupResource]value.Transformer
+	closeTransformers    context.CancelFunc
+}
+
+type notificationChannel struct {
+	once sync.Once
+	ch   chan struct{}
+}
+
+// NewDynamicTransformers initializes DynamicTransformers when api-server starts.
```

Comment:
Remove this comment, it doesn't contain meaningful information.

Reply:
updated to keep linters happy